### PR TITLE
Sidebar menu and labels styling adjustment

### PR DESCRIPTION
**Pull request by floreks**
I've just reverted bottom padding for the sidebar subline, reduced a bit bottom margin between labels and fixed background color.

zrzut ekranu z 2016-01-11 10-04-04

### CODE REVIEW

File: src/app/frontend/replicasetdetail/replicasetdetail.scss [hunk 1]
```diff
@@ -78,6 +78,7 @@ $replicasetdetails-sidebar-width: 315px;
 .kd-replicasetdetail-sidebar-subline {
   color: $muted;
   font-size: $caption-font-size-base;
+  padding-bottom: 8px;
```
Comment (bryk) [inline src/app/frontend/replicasetdetail/replicasetdetail.scss]: 8px is $baseline-grid. Should you use it? I think so

Comment (floreks) [inline src/app/frontend/replicasetdetail/replicasetdetail.scss]: Done.

**Comment by bryk** [comment]
Just a quick comment. PTAL

**Comment by floreks** [comment]
It's done.

**Comment by bryk** [comment]
LGTM

**Pull request merged**